llvm/include/llvm/Transforms/Utils/LoopUtils.h

```diff
@@ -411,8 +411,8 @@ Value *createSimpleReduction(IRBuilderBase &B, Value *Src,
                              RecurKind RdxKind);
 /// Overloaded function to generate vector-predication intrinsics for
 /// reduction.
-Value *createSimpleReduction(VectorBuilder &VB, Value *Src,
-                             const RecurrenceDescriptor &Desc);
+Value *createSimpleReduction(VectorBuilder &VB, Value *Src, RecurKind RdxKind,
```

Comment:
The loop utils changes can be split off?

Reply:
Done in #131336

Reply:
Thanks! I originally meant to split off the changes to update the interfaces in LoopUtils in general to not take the recurrence descriptor directly, which could also be done separately.

Reply:
Oh whoops, I'd still be happy to split those off if you'd prefer?

Reply:
Yep I think that would make sense.

Reply:
These changes should be done in main now after #132017